treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -205,11 +211,12 @@ def _ignore_sensitive_fields(self, table_name, model_values):
         """
         UPSTREAM_DATABASE_URL credentials don't generally have read rights on
         the auth_user table (which contains users' password hashes).
-        Simply setting `classifier = None` won't cause access errors.
+        Simply setting `model.sensitive_field = None` won't cause access errors.
         """
-        if table_name == 'performance_alert':
+        if table_name == 'performance_alert' or table_name == 'performance_alert_summary':
             for model in model_values:
                 model.classifier = None
+                model.assignee = None
```

Comment:
Alerts don't have `assignee` & alert summaries don't have `classifier`.

Reply:
Indeed, this is not convenient, but was a way of avoiding iterating twice over the `movel_values`.

Reply:
I changed the code, hopefully this is better.